docs/podman-container-install.1.md

```diff
@@ -0,0 +1,72 @@
+% PODMAN(1) Podman Man Pages
+% Dan Walsh
+% January 2015
+# NAME
+podman-container-install - Execute Image Install Method
+
+# SYNOPSIS
+**podman container install**
+[**-h**|**--help**]
+[**--display**]
+[**-n**][**--name**[=*NAME*]]
+[**--rootfs**=*ROOTFS*]
+[**--set**=*NAME*=*VALUE*]
+[**--storage**]
+[**--system-package=auto|build|yes|no**]
+[**--system**]
+IMAGE [ARG...]
+
+# DESCRIPTION
+**podman container install** attempts to read the `LABEL INSTALL` field in the container
+IMAGE, if this field does not exist, `podman container install` will install the IMAGE.
```

Comment:
Again, what does "install the image" mean? Just pull it?

Reply:
fixed